doc/old_and_new_proposal.md

```diff
@@ -264,12 +264,12 @@ use case.
 ```xml
 <partitioning>
   <proposal>
-    <lvm config:type="boolean">false</default_lvm>
-    <encrypt config:type="boolean">false</default_encrypt>
+    <lvm config:type="boolean">false</lvm>
+    <encrypt config:type="boolean">false</encrypt>
     <windows_delete_mode>all</windows_delete_mode>
-    <linux_delete>ondemand</linux_delete_mode>
+    <linux_delete_mode>ondemand</linux_delete_mode>
     <other_delete_mode>ondemand</other_delete_mode>
-    <lvm_vg_strategy_mode>use_available</lvm_vg_strategy>
+    <lvm_vg_strategy_mode>use_available</lvm_vg_strategy_mode>
```

Comment:
Actually the fix would be the other way around. The correct one is `lvm_vg_strategy` (without the "mode" part)

Reply:
Ohh, you are right :)